pyworkflow/em/protocol/protocol_align_movies.py

```
@@ -243,8 +243,8 @@ def _validate(self):
         # self.inputMovies.get().close()
         # frames = movie.getNumberOfFrames()
 
-	if self.inputMovies.get() is None:
-            errors.append("Input movies is None")
```

I think we should report the error even if we do not proceed with other validations